tests/mlir_printer_test.py

```diff
@@ -73,7 +73,7 @@ class ParamAttrWithCustomFormat(ParametrizedAttribute):
     param1: ParameterDef[ParamAttr]
 
     def print_parameters(self, printer: Printer) -> None:
-        printer.print(f"~~")
+        printer.print(f'{"~~"}')
```

Comment:
```suggestion
        printer.print("~~")
```
But I would guess that a rebase would fix that?